connectors/src/connectors/github/temporal/activities.ts

```diff
@@ -941,6 +968,14 @@ export async function githubCodeSyncActivity({
   githubCodeRepository.lastSeenAt = codeSyncStartedAt;
   await githubCodeRepository.save();
 
+  // Add as dataSource folder
+  await upsertFolderNode({
+    dataSourceConfig,
+    folderId: githubCodeRepository.repoId,
+    title: githubCodeRepository.repoName,
+    parents: [githubCodeRepository.repoId],
```

Comment:
don't forget, parents[0] === folder_id as you changed in `connectors/src/connectors/github/lib/hierarchy.ts`

Reply:
In this case we are uploading a node for the repository itself, which is at root level, that's why it is its own id